tests/functional/posting_test.php

```diff
@@ -71,4 +71,85 @@ public function test_html_entities()
 		$crawler = self::request('GET', "viewtopic.php?t={$post['topic_id']}&sid={$this->sid}");
 		$this->assertContains('&#128512;', $crawler->text());
 	}
+
+	/**
+	* @testdox max_quote_depth is applied to the text populating the posting form
```

Comment:
A description of the test as a declarative sentence, because it's hard to tell what is being tested just by looking at the code. It can also be used to generate nice checklists like [this one](https://github.com/s9e/TextFormatter/blob/master/docs/testdox.txt).